Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInput.cs

```diff
@@ -40,7 +43,7 @@ namespace UnityEngine.InputSystem
     /// <remarks>
     /// PlayerInput is a high-level wrapper around much of the input system's functionality
     /// which is meant to help getting set up with the new input system quickly. It takes
-    /// care of <see cref="InputAction"/> bookkeeping and has a custom UI to help
+    /// care of <see cref="InputAction"/> bookkeeping and has a custom UI(Requires UI package) to help
```

Comment:
Nitpick... ` (requires UI package)`

Reply:
Actually, on second thought, given the ambiguities of what "UI" may refer to, probably better to explicitly use the name of the package here (`requires the "Unity UI" package`).